htdocs/commande/class/commande.class.php

```diff
@@ -2194,6 +2194,10 @@ public function deleteline($user = null, $lineid = 0)
 					// For triggers
 					$line->fetch($lineid);
 
+					// Memorize previous line for triggers
+					$staticline = clone $line;
+					$line->oldline = $staticline;
```

Comment:
There is a property dedicated for this on each object that is ->oldcopy. Can you use oldcopy instead of oldline ?

Reply:
of course, but please note that everywhere else oldline is used...

Reply:
@eldy i insist that oldline is used everywhere in this scripts and the others (and surely by external modules also)

Reply:
Right, sorry. So it's ok with oldline instead of oldcopy when it is a line.